app/controllers/v1/blog_posts_controller.rb

```diff
@@ -1,9 +1,15 @@
 class V1::BlogPostsController < V1::BaseController
   def index
-    render json: BlogFetcher.feed(:recent)
+    render json: BlogFetcher.feed(param: :recent, reset: parsed_reset)
   end
 
   def press_releases
-    render json: BlogFetcher.feed(:press_releases)
+    render json: BlogFetcher.feed(param: :press_releases, reset: parsed_reset)
+  end
+
+  protected
+
+  def parsed_reset
+    params[:reset].presence == "1"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.